src/compiler/types.ts

```diff
@@ -896,6 +896,7 @@ export const enum ModifierFlags {
     Modifier = All & ~Decorator,
 }
 
+/** @internal */
 export const enum JsxFlags {

```

Comment:
This is used once, but I feel like this shouldn't be exported: https://github.com/search?q=%2F%5Cbts%5C.JsxFlags%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code